### PR TITLE
Adding a merchandise link to the marketplace selector

### LINKED ISSUE #8265: Add link to merchandise page in Market
We realized that the merchandise page is only linked in the footer, we should add a link in the market page.

schermata_2016-12-07_alle_20_36_22

### PR DESCRIPTION
**Pull request by Hus274**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #8265

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Adding a merchandise link to the right of the marketplace selector that brings the user to the static merchandise page.

screen shot 2016-12-08 at 8 19 52 am

### CODE REVIEW

**Comment by paglias** [comment]
@Hus274 thanks for this! Can the link have a style similar to that of the other menu items?

File: website/views/options/inventory/menu.jade [hunk 1]
```diff
@@ -20,6 +20,9 @@ ul.options-menu
   li(ng-class="{ active: $state.includes('options.inventory.seasonalshop') }")
     a(ui-sref='options.inventory.seasonalshop')
       =env.t('seasonalShop')
+  div.pull-right
```
Comment (crookedneighbor) [inline website/views/options/inventory/menu.jade]: The div is unnecessary. Putting in a class or id without an element assumes a div.

**Review by crookedneighbor — APPROVED** [review]
:+1:

**Comment by Hus274** [comment]
How about this?

screen shot 2016-12-09 at 7 31 29 am

**Comment by Hus274** [comment]
The tests seem to be failing for a completely unrelated reason to this change.

**Pull request merged**